tasks/01-strings.js

```diff
@@ -13,7 +13,8 @@
 //  If you get a number with .0 on the end, return the integer value (e.g. return "4" rather than "4.0").
 //  If the number is 0, 0.0, 000, 00.00, etc... return "0".
 function removeLeadingTrailing(n) {
-  throw new Error('Not implemented');
+  const answer = parseFloat(n).toString();
```

Comment:
It isn't necessary to create an extra variable. You can return the value immediately